### ISSUE TITLE
Process finished with exit code 139 (interrupted by signal 11: SIGSEGV) 

### ISSUE THREAD

**Issue by meanmee** [apache/incubator-mxnet#3661]
I am building a CNN+RNN model for speech recognition. But it seems a lot problems with MXNet's rnn API?
I followed the rnn example and trained my network finally after someone's help.
But when it comes to the predict step, some issues occur:
Here are my codes:

x = mx.io.NDArrayIter(data=Pxxs_arr, label=None, batch_size=10, shuffle=False, last_batch_handle='pad')
pred = self.model.predict(x)

hint: Pxx.shape = (10, 1, x, y)
(batch_size, chnnel, w, h)

Then:
Process finished with exit code 139 (interrupted by signal 11: SIGSEGV)

Since My codes is work pretty well in my single cnn network experienments:
pred = self.model.predict(Pxx)

hint: Pxx is a numpy array with shape(1, 1, x, y)

So, what's the true skills of MXNet's RNN API?

here are my codes of the symbol(CNN + RNN):

`data = mx.symbol.Variable('data')
rnn_h_init = mx.sym.SwapAxis(mx.sym.Variable('LSTM_init_h'),
dim1=0, dim2=1)
rnn_c_init = mx.sym.SwapAxis(mx.sym.Variable('LSTM_init_c'),
dim1=0, dim2=1)
rnn_params = mx.sym.Variable('LSTM_bias')

print_infer_shape(data)

conv1 = mx.symbol.Convolution(data=data, kernel=(3, 5), num_filter=64, stride=(1, 4))
bn1 = mx.symbol.BatchNorm(data=conv1, fix_gamma=False, momentum=0.9, eps=1e-5)
relu1 = mx.symbol.Activation(data=bn1, act_type="relu")
pool1 = mx.symbol.Pooling(data=relu1, pool_type="max",
                          kernel=(2, 2), stride=(2, 2))

print_infer_shape(pool1)
conv2 = mx.symbol.Convolution(data=pool1, kernel=(3, 3), num_filter=128, stride=(1, 2))
bn2 = mx.symbol.BatchNorm(data=conv2, fix_gamma=False, momentum=0.9, eps=1e-5)
relu2 = mx.symbol.Activation(data=bn2, act_type="relu")
pool2 = mx.symbol.Pooling(data=relu2, pool_type="max",
                          kernel=(2, 2), stride=(2, 2))
print_infer_shape(pool2)

conv3 = mx.symbol.Convolution(data=pool2, kernel=(3, 3), num_filter=256, stride=(1, 2))
bn3 = mx.symbol.BatchNorm(data=conv3, fix_gamma=False, momentum=0.9, eps=1e-5)
relu3 = mx.symbol.Activation(data=bn3, act_type="relu")
pool3 = mx.symbol.Pooling(data=relu3, pool_type="max",
                          kernel=(1, 2), stride=(1, 2))
print_infer_shape(pool3)

conv4 = mx.symbol.Convolution(data=pool3, kernel=(2, 2), num_filter=512, stride=(1, 1))
bn4 = mx.symbol.BatchNorm(data=conv4, fix_gamma=False, momentum=0.9, eps=1e-5)
relu4 = mx.symbol.Activation(data=bn4, act_type="relu")
print_infer_shape(relu4)

reshape1 = mx.symbol.Reshape(data=relu4, shape=(10, 512, -1))
swapaxis1 = mx.symbol.SwapAxis(data=reshape1, dim1=0, dim2=1)
swapaxis2 = mx.symbol.SwapAxis(data=swapaxis1, dim1=0, dim2=2)

bilstm = mx.symbol.RNN(data=swapaxis2, bidirectional=False, num_layers=3, mode='lstm',
                       state_size=128, state_outputs=False, state=rnn_h_init,
                       state_cell=rnn_c_init, parameters=rnn_params)
print_infer_shape(bilstm)
swapaxis3 = mx.symbol.SwapAxis(data=bilstm, dim1=0, dim2=1)
print_infer_shape(swapaxis3)
mean_state = mx.symbol.sum(data=swapaxis3, axis=1) / 177.
print_infer_shape(mean_state)
fc1 = mx.symbol.FullyConnected(data=mean_state, num_hidden=num_classes)
neonscript_withrnn = mx.symbol.SoftmaxOutput(data=fc1, label=label, name='softmax')`

**Comment by yajiedesign** [comment]
This issue is closed due to lack of activity in the last 90 days. Feel free to reopen if this is still an active issue. Thanks!
